doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+__children["RaySwitchShader"]["parameters"]["specular_reflection"].setInput( __children["TraceSet1"]["out"] )
+__children["RaySwitchShader"]["parameters"]["specular_transmission"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["volume"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["__uiPosition"].setValue( imath.V2f( -9.90797043, -103.52475 ) )
+__children["ShaderAssignment"]["enabled"].setValue( False )
+__children["ShaderAssignment"]["in"].setInput( __children["ShaderAssignment1"]["out"] )
+__children["ShaderAssignment"]["filter"].setInput( __children["PathFilter"]["out"] )
+__children["ShaderAssignment"]["shader"].setInput( __children["RaySwitchShader"]["out"] )
+__children["ShaderAssignment"]["__contextCompatibility"].setValue( False )
+__children["ShaderAssignment"]["__uiPosition"].setValue( imath.V2f( 7.23167801, -103.52475 ) )
+__children["PathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/world/geo/plane' ] ) )
+__children["PathFilter"]["__uiPosition"].setValue( imath.V2f( 18.0980415, -97.9078064 ) )
+__children["StandardSurface"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface"]["__uiPosition"].setValue( imath.V2f( -39.750576, -99.9247513 ) )
+__children["StandardSurface2"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface2"]["__uiPosition"].setValue( imath.V2f( -39.509861, -115.691742 ) )
+__children["TraceSet1"]["parameters"]["passthrough"].setInput( __children["StandardSurface2"]["out"] )
+__children["TraceSet1"]["parameters"]["trace_set"].setValue( 'spheres' )
+__children["TraceSet1"]["parameters"]["inclusive"].setValue( False )
+__children["TraceSet1"]["__uiPosition"].setValue( imath.V2f( -24.7410622, -115.691742 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop4"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop4"]["title"].setValue( 'Note: New objects' )
+__children["Backdrop4"]["description"].setValue( 'Nodes such as Sphere, Plane, lights, etc have a Sets plug. This can be used to put them in one or more sets at creation without needing the separate Set node.' )
+__children["Backdrop4"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 20, 7.88689423 ) ) )
+__children["Backdrop4"]["__uiPosition"].setValue( imath.V2f( 39.2844086, -35.3775558 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop5"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop5"]["title"].setValue( 'Note: Set names' )
```

Comment:
### Backdrop title
Note: Referring to the trace set

Reply:
Same as the other note, re. title width.